lib/active_merchant/billing/gateways/bambora_apac.rb

```diff
@@ -82,6 +82,26 @@ def void(money, authorization, options={})
         end
       end
 
+      def verify(credit_card, options={})
+        MultiResponse.run(:use_first_response) do |r|
+          r.process { purchase(200, credit_card, options) }
```

Comment:
I assume 200 is it the minimum? If not, try 0.

Reply:
I wanted to, but I could not get 100 or 0 to work. Voids only work on purchases, not auths, and it won't accept a $0 purchase. It seems that 100 is an amount that generates an error for tests ("declined_message"=>"Refer Card Issuer"), though that did not seem apparent in the docs. https://dev-apac.bambora.com/checkout/guides/get-up-and-running/test-your-setup

Reply:
🤔 I wonder if an un-voided auth is preferable...

Reply:
I was having the same thought, it would be lame if the void fails, then there are random $2 purchases 😬

Reply:
Updated to a 0 un-voided auth.

Reply:
@molbrown Sorry for leaving this out initially, but there may be other reasons un-voided auths are undesirable. It should be fine if they are automatically canceled after a short time, but I can't tell if this is the case from their Docs. Any insight on that?

Reply:
I have contacted Bambora support about this, and have decided to remove `verify` from this commit and proceed without it. It will be added in once there is more information about how to best implement.